### PR TITLE
TASK: Add TYPO3.TypoScript:Debug to Reference

### PR DESCRIPTION
**Pull request by dlubitz**

### CODE REVIEW

File: TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst [hunk 1]
```diff
@@ -169,6 +169,34 @@ overriding the ``@position`` property.
...
+:plaintext: (boolean) If set true, the result will be shown as plaintext
+:[key]: (mixed) A nested definition (simple value, expression or object), ``[key]`` will be used as key for the resulting output
+
+Example::
+
+  debugObject = Debug {
+        title = 'Debug of hello world'
+
+        # If only the value-key is given it is debugged directly,
+        # otherwise all keys except title an plaintext are debugged.
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst]: I'd suggest to quote "value"-key, "title" and "plaintext" to make it more clear

File: TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst [hunk 2]
```diff
@@ -169,6 +169,34 @@ overriding the ``@position`` property.
...
+        value = "hello neos world"
+
+        # Additional values for debugging
+        documentTitle = ${q(documentNode).property('title')}
+        documentPath = ${documentNode.path}
+  }
+  
+  # the value of this object is the formated debug output of all keys given to the object
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst]: formatted

**Comment by dlubitz** [comment]
@aertmann Thanks. Is done.

File: TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst [hunk 3]
```diff
@@ -169,6 +169,34 @@ overriding the ``@position`` property.
...
+:plaintext: (boolean) If set true, the result will be shown as plaintext
+:[key]: (mixed) A nested definition (simple value, expression or object), ``[key]`` will be used as key for the resulting output
+
+Example::
+
+  debugObject = Debug {
+        title = 'Debug of hello world'
+
+        # If only the "value"-key is given it is debugged directly,
+        # otherwise all keys except "title" an "plaintext" are debugged.
```
Comment (hlubek) [inline TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst]: s/an/and/

**Pull request merged**